docs/UserGuide.adoc

```diff
@@ -72,6 +72,12 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+
+== Sorting all persons : `sort`
+
+Sort addressbook name list by alphabetical order. +
```

Comment:
You could possibly give more information about your sort command, since in your case, after sorting, the order of the people in the addressbook will be permanently changed.

Reply:
Hmmm, reasonable. I should make a copy to modify.